### ISSUE TITLE
Problem running docker on Debian 11?

### ISSUE THREAD

**Issue by gwhizoftv** [cdot/Xanado#56]
Issue:
Appears you need to edit config.json in the docker before you can run it. The directions for running the docker aren't documented ? 

Steps:
I ran 

- docker pull ghcr.io/cdot/xanado:release-2.1.1

then tried 

- docker run db04ea431ece

and got the error:

`admin@ip-172-31-24-191:~/scrabble$ docker run db04ea431ece
(node:1) UnhandledPromiseRejectionWarning: AssertionError [ERR_ASSERTION]: No config.defaults, see example-config.json for requirements
    at new Server (/app/js/server/Server.js:67:16)
    at /app/server.js:101:17
(node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:1) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
`

**Comment by gwhizoftv** [comment]
HI, I upgraded to node 18 and it changed.
Error on node_modules/cldrpluralruleparser
so I:
 cd node_modules/cldrpluralruleparser ; npm install

Tried again, running got 
```
node --trace-warnings server.js
node:internal/modules/cjs/loader:959
  throw err;
  ^


Error: Cannot find module '/Users/MGW/server.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Module._load (node:internal/modules/cjs/loader:804:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

**Comment by gwhizoftv** [comment]
Since I couldn't build I tried the docker;

I ran 👍 
docker pull ghcr.io/cdot/xanado:release-2.1.1
then tried 
docker run db04ea431ece
and got the error:
`admin@ip-172-31-24-191:~/scrabble$ docker run db04ea431ece
(node:1) UnhandledPromiseRejectionWarning: AssertionError [ERR_ASSERTION]: No config.defaults, see example-config.json for requirements
    at new Server (/app/js/server/Server.js:67:16)
    at /app/server.js:101:17
(node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:1) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
`
